packages/@aws-cdk/pipelines/lib/actions/deploy-cdk-stack-action.ts

```diff
@@ -1,11 +1,11 @@
+import * as path from 'path';
```

Comment:
This was a part of using `--fix` to clean up linting issues. It seems it had a bad import order throwing a warning beforehand anyway.